integration_tests/seeds/dag/dag_seeds.yml

```diff
@@ -2,27 +2,18 @@ version: 2
 
 seeds:
   - name: test_fct_multiple_sources_joined
-    config:
-      column_types:
-        CHILD: varchar
-        SOURCE_PARENTS: varchar
     tests:
       - dbt_utils.equality:
+          name: equality_fct_multiple_sources_joined
           compare_model: ref('fct_multiple_sources_joined')
           compare_columns:
             - child
             - source_parents
 
   - name: test_fct_direct_join_to_source
-    config:
-      column_types:
-        PARENT: varchar
-        PARENT_RESOURCE_TYPE: varchar
-        CHILD: varchar
-        CHILD_RESOURCE_TYPE: varchar
-        DISTANCE: int
     tests:
       - dbt_utils.equality:
+          name: equality_fct_direct_join_to_source
```

Comment:
purely out of curiosity, what are these configs necessary?

Reply:
I have added those to avoid our tests being called `equality_test_fct_direct_join_to_source_col1_col2_col3_...fct_direct_join_to_source...`, making it easier to read and look for compiled SQL.